[2022-08-03 17:13:54,957] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ltv-pipeline.decompress-file manual__2022-08-03T17:13:42.739593+00:00 [queued]>
[2022-08-03 17:13:54,979] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ltv-pipeline.decompress-file manual__2022-08-03T17:13:42.739593+00:00 [queued]>
[2022-08-03 17:13:54,981] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 17:13:54,982] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-03 17:13:54,984] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 17:13:55,001] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): decompress-file> on 2022-08-03 17:13:42.739593+00:00
[2022-08-03 17:13:55,010] {standard_task_runner.py:52} INFO - Started process 3143 to run task
[2022-08-03 17:13:55,015] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ltv-pipeline', 'decompress-file', 'manual__2022-08-03T17:13:42.739593+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/dag-test.py', '--cfg-path', '/tmp/tmpqrobzecm', '--error-file', '/tmp/tmpysb46e5c']
[2022-08-03 17:13:55,019] {standard_task_runner.py:80} INFO - Job 92: Subtask decompress-file
[2022-08-03 17:13:55,146] {task_command.py:371} INFO - Running <TaskInstance: ltv-pipeline.decompress-file manual__2022-08-03T17:13:42.739593+00:00 [running]> on host 3b0f05b8acbc
[2022-08-03 17:13:55,281] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ltv-pipeline
AIRFLOW_CTX_TASK_ID=decompress-file
AIRFLOW_CTX_EXECUTION_DATE=2022-08-03T17:13:42.739593+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-03T17:13:42.739593+00:00
[2022-08-03 17:13:55,297] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-03 17:13:55,309] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'gunzip files/vehicle_challenge.csv.gz']
[2022-08-03 17:13:55,358] {subprocess.py:85} INFO - Output:
[2022-08-03 17:13:55,432] {subprocess.py:92} INFO - gzip: files/vehicle_challenge.csv.gz: No such file or directory
[2022-08-03 17:13:55,441] {subprocess.py:96} INFO - Command exited with return code 1
[2022-08-03 17:13:55,534] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 195, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2022-08-03 17:13:55,597] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=ltv-pipeline, task_id=decompress-file, execution_date=20220803T171342, start_date=20220803T171354, end_date=20220803T171355
[2022-08-03 17:13:55,662] {standard_task_runner.py:97} ERROR - Failed to execute job 92 for task decompress-file (Bash command failed. The command returned a non-zero exit code 1.; 3143)
[2022-08-03 17:13:55,735] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-08-03 17:13:55,839] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
